Esri/arcgis-maps-sdk-kotlin-samples
# {DO NOT MERGE] Tests Shubham's MainActivity.kt template for Compose project

# This PR should not be merged

Purpose of this PR is to test the MainActivity.kt file that Shubham provided to use as a template for creating new map projects that use Jetpack Compose.

All the project's source code is in `MainActivity.kt`, instead of re-factoring into `MainScreen.kt` and `MapViewModel.kt` files.

To test this project:

(1) Open the `arcgis-maps-sd-kotlin-samples` project in Android Studio

(2) Run the `shubham-template-mobile-map-package` project. I wrote this code using a MapViewModel.
     EXPECTED RESULTS: A map of Yellowstone National Park should display against a gray grid.

(3) Comment out the current code: **lines 47 - 104**

(4) Uncomment lines **109 - 160**. Shubham wrote this code for me to use as a template when creating Compose-based projects. Note that it intentionally avoids creating a MapViewModel.
    EXPECTED RESULTS: A map of Yellowstone National Park should display against a gray grid.
    ACTUAL RESULTS: Map view displays a gray grid, but no map is visible. Every 12-15 seconds, the Yellowstone map flashes _very_ briefly in the map view, and then immediately disappears.
